src/app/_directives/autosize.directive.ts

```diff
@@ -9,7 +10,7 @@ export class AutosizeDirective implements AfterViewInit {
   private _minHeight: number;
   private _maxHeight: number;
   private _clientWidth: number;
-  private _rows: number = 0;
+  private _rows = 0;
```

Comment:
why?

Reply:
If you declare and assign in the same line, the compile can infer the type automatically. The linter suggested that change but I can add it back.